modules/relay/src/main/RelayTourRepo.scala

```diff
@@ -15,6 +15,17 @@ final private class RelayTourRepo(val coll: Coll)(using Executor):
   def countByOwner(owner: UserId): Fu[Int] =
     coll.countSel(selectors.ownerId(owner))
 
+  def subscribers(tid: RelayTour.Id): Fu[Set[UserId]] =
+    coll.distinctEasy[UserId, Set]("subscribers", $id(tid))
+
+  def setSubscribed(tid: RelayTour.Id, uid: UserId, isSubscribed: Boolean): Funit =
+    coll.update
+      .one($id(tid), if isSubscribed then $push("subscribers" -> uid) else $pull("subscribers" -> uid))
```

Comment:
`$push` -> `$addToSet`

Reply:
yeah that avoids dupes so is better, $pull does remove all instances though right?